.zenodo.json

```diff
@@ -0,0 +1,22 @@
+  {
+    "license": "other-open", 
```

Comment:
Shouldn't this be MIT?

Reply:
still in draft because I've been trying to find their list of possible enum values for this field 😄

Reply:
```suggestion
    "license": "mit-license", 
```